homeassistant/components/glances/config_flow.py

```diff
@@ -36,7 +35,7 @@
         vol.Optional(CONF_USERNAME): str,
         vol.Optional(CONF_PASSWORD): str,
         vol.Required(CONF_PORT, default=DEFAULT_PORT): int,
-        vol.Required(CONF_VERSION, default=DEFAULT_VERSION): int,
+        vol.Required(CONF_VERSION, default=DEFAULT_VERSION): vol.In([2, 3]),
```

Comment:
I think keeping the constant, and using it here is clearer:
```suggestion
        vol.Required(CONF_VERSION, default=DEFAULT_VERSION): vol.In(SUPPORTED_VERSIONS),
```

Reply:
Done